src/mod/say/mod_say_fr/mod_say_fr.c

```diff
@@ -119,7 +119,7 @@ static switch_status_t play_group(switch_say_method_t method, switch_say_gender_
 		}
 	}
 
-	if (c || ((ftdNumber == 1 && (a || b || c)) && (a && (b || c)))) {
+	if (c || ((ftdNumber == 1 && (a || b || c)) && (a && (b || c))) || (a==0 && b==1 && c==0) ) {
```

Comment:
use FS indentation (use spaces), eg: (a == 0 && b == 1 && c == 0)

Reply:
Is that better to you ?